Appease pyre

clee2000 opened the pull request:
Summary: appease pyre


Differential Revision: D58560545

mergennachin commented on exir/sym_util.py:
```diff
@@ -73,14 +77,14 @@ def eval_shape(shape):
 
 
-def collect_free_symbols(shape) -> Set[sympy.Symbol]:
+def collect_free_symbols(shape) -> Set[sympy.Symbol]:  # pyre-ignore[2]
```
can you annotate shape with Union[int, torch.SymInt]?

clee2000 replied:
Is this true for all the functions?

Commit:
Appease pyre (#3979)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/3979

Remove pyre annotations that are no longer needed

Add some ignores so I can add # pyre-strict, I don't actually know what the types of these things are

Reviewed By: mergennachin

Differential Revision: D58560545
